# Actividad 1 - Estudio sobre sedentarismo y ejercicio

| Tabla cruzada  | Con sobrepeso | Sin sobrepeso | Total |
|----------------|---------------|---------------|-------|
| Sedentarios    | 100           | 200           | 300   |
| No Sedentarios | 50            | 650           | 700   |
| Total          | 150           | 850           | 1000  |

In [1]:
valores_observados <- data.frame(
    "con_sobrepeso" = c(100, 50),
    "sin_sobrepeso" = c(200, 650)
)
rownames(valores_observados)<-c("Sedentarios", "No_sedentarios")
valores_observados

con_sobrepeso sin_sobrepeso
Sedentarios    100           200          
No_sedentarios  50           650

Hacemos la prueba de independencia de Pearson

In [2]:
# Cuando correct=T, chisq hace una correción de continuidad de Yates
# Esto cuando es una matriz de 2x2
chi2<-chisq.test(valores_observados, correct=FALSE)
#Valores esperados
#chi2$expected
# Imprimimos el estadístico de prueba
chi2

#help(chisq.test)


	Pearson's Chi-squared test

data:  valores_observados
X-squared = 112.98, df = 1, p-value < 2.2e-16


En general, se aplica la _corrección de Yates_ o también conocida como corrección por continuidad cuando se aproxima una variable discreta a una distribución continua. La corrección consiste en añadir y substraer 0,5 a la variable en cuestión. Por ejemplo, obtener 3 caras al lanzar una moneda es una medida discreta (nominal) que se ajusta a la distribución binomial. Mientras que si la aproximáramos a la distribución normal, su valor oscilará entre 2,5 y 3,5. 

Hacemos la prueba de la máxima verosimilitud.

In [3]:
library(DescTools)
GTest(valores_observados)


	Log likelihood ratio (G-test) test of independence without correction

data:  valores_observados
G = 103.26, X-squared df = 1, p-value < 2.2e-16


Hacemos las pruebas de riesgo.

In [4]:
library(epiR)
#epi.2by2(as.matrix(valores_observados), conf.level=.95, method="cohort.count")
pruebas<-epi.2by2(as.matrix(valores_observados), conf.level=.95, method="cohort.count")

Loading required package: survival
Package epiR 1.0-2 is loaded
Type help(epi.about) for summary information




In [5]:
pruebas

             Outcome +    Outcome -      Total        Inc risk *        Odds
Exposed +          100          200        300             33.33      0.5000
Exposed -           50          650        700              7.14      0.0769
Total              150          850       1000             15.00      0.1765

Point estimates and 95% CIs:
-------------------------------------------------------------------
Inc risk ratio                               4.67 (3.42, 6.37)
Odds ratio                                   6.50 (4.47, 9.45)
Attrib risk *                                26.19 (20.53, 31.86)
Attrib risk in population *                  7.86 (4.94, 10.78)
Attrib fraction in exposed (%)               78.57 (70.74, 84.30)
Attrib fraction in population (%)            52.38 (41.05, 61.53)
-------------------------------------------------------------------
 Test that odds ratio = 1: chi2(1) = 112.979 Pr>chi2 = < 0.001
 Wald confidence limits
 CI: confidence interval
 * Outcomes per 100 popula

De lo anterior se obtiene:
  - Hay evidencia estadísticamente significativa como para decir que hay una relación entre el sobrepeso y el sedentarismo de la gente.
  - Del riesgo relativo (RR) se observa que existe una asociación positiva entre el sedentarismo y el desarrollo de sobrepeso, por lo cual una persona con vida sedentaria tiene 4.6 veces más de riesgo de desarrollar sobrepeso que si llevase una vida activa.
  - Del riesgo atribuido(RA) se intuye que llevar una vida sedentaria incrementa el riesgo de desarrollar sobrepeso 26%.
  - Para terminar, de la razón de momios se obtiene que sí hay una asociación positiva entre las variables y que es 6.5 veces más probable que aquellas personas que tienen sobrepeso lo hayan desarrollado por llevar una vida sedentaria que de no haberla llevado. Nótese que el intervalo de confianza no queda en el área menor al 0 que haga rechazar esta afirmación.

